tests/Model/Generated/Endpoint/SessionTest.php

```diff
@@ -31,6 +36,6 @@ public function testDeleteSession()
      */
     public static function tearDownAfterClass()
     {
-        static::ensureApiContextValid();
+        unlink(static::FILENAME_CONTEXT_CONFIG);
```

Comment:
Oops! I must have missed it at some earlier point. Let us please use `self::` for constants :)